src/main_app/jobs_workers/admin_jobs_workers/slugs_helpers.py

```
@@ -0,0 +1,28 @@
+    original_chart_url = metadata.get("chart", {}).get("originalChartUrl", "")
+    if not original_chart_url or "/grapher/" not in original_chart_url:
+        return False
+
+    original_slug = original_chart_url.split("/grapher/", maxsplit=1)[1].split("?")[0]
+    if original_slug == _slug_to_check:
+        return False
+
+    try:
+        add_new_slug_redirect(slug=_slug_to_check, redirect_to=original_slug)
+        return True
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)